app/services/recording_sync.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class RecordingSync
+  def initialize; end
+
+  def recording_resync(user:)
+    bbb_api = BigBlueButtonApi.new
+    rooms = user.rooms
+
+    # Get the meeting_id of all the current user's rooms
+    meeting_ids = rooms.map(&:meeting_id)
+
+    # Clear current recordings
+    Recording.destroy_by(room_id: rooms.map(&:id))
+
+    # Get recordings
```

Comment:
There's too many comments here - drop all the comments that are straight forward and only leave comments for things that are either odd or complex